cypress/e2e/universal-search.test.ts

```diff
@@ -1,13 +1,17 @@
 import { getTestSelector } from '../utils'
 
 describe('Universal search bar', () => {
-  before(() => {
-    cy.visit('/')
+  function openSearch() {
     cy.get('[data-cy="magnifying-icon"]')
       .parent()
       .then(($navIcon) => {
         $navIcon.click()
       })
```

Comment:
In general, you can just invoke `click` on the yielded subject: `.parent().click()`.